atomic_physics/utils.py

```diff
@@ -108,7 +108,7 @@ def ac_zeeman_shift(atom: ap.Atom, state: int, f_RF: float):
 
     E = atom.E[states]
     M = atom.M[states]
-    R = atom.M1[states]
+    R = atom.M1[states, states]
```

Comment:
I'm not sure this is right. in the following lines we do:
```python3
rabi = R / consts.hbar
w = rabi[state, _state][0]
```
As we set `state -= states.start`, shouldn't the first axis of `R` have the orignial dimension?

The code reads more like we'd want something like:
```python3
R = atom.M1[:, states]
```
Though I haven't checked the atomic physics on that.

Reply:
`w = rabi[state, _state][0]` yeah, that's not needed, not sure why it's there. Should just be `w = rabi[state, _sate]`